demos/tests/cases.py

```diff
@@ -222,6 +222,20 @@ def device_cases(*args):
             }),
         ],
     )),
+    
+    PythonDemo(name='text_detection_demo', test_cases=combine_cases(
```

Comment:
I won't pass, because testing environment doesn't have a display. The common solution is to add `--no_show` key to disable `imshow` and `waitKey` in the demo and adding that key to a test. Will you do it, please?

Reply:
Well I did it, but I am like blindly walking here. Not sure that it should be `'--no_show': None` and not `-no_show` or `'-no_show': None` e.g.

Reply:
The demo should implement this key, disabling `imshow` and `waitKey`. The key should be `--no_show`, `--` comes from a convention that it should be `--` for long keys for python. For the test it should be `--no_show: None`. `None` means there is no value with that key.

Reply:
Aha, I thought so, but did add it to segmentation python demo by mistake

Reply:
The demo should implement this key, disabling `imshow` and `waitKey`.